yb-voyager/src/migassessment/sizing.go

```diff
@@ -1471,10 +1472,20 @@ func filterRedundantIndexes(sourceIndexMetadata []SourceDBMetadata, redundantInd
 			} else {
 				tableName = indexMetadata.ParentTableName.String // Fallback to full string
 			}
+		} else {
+			// If parent table name is not valid, include the index (can't match against redundant list)
+			filteredIndexes = append(filteredIndexes, indexMetadata)
+			continue
 		}
 
-		// Create key to check against redundant indexes
-		key := fmt.Sprintf("%s.%s.%s", indexMetadata.SchemaName, tableName, indexMetadata.ObjectName)
+		// Create a temporary RedundantIndexesInfo to use the same helper method for consistent formatting
+		tempRedundantInfo := utils.RedundantIndexesInfo{
+			DBType:              *sourceDBType,
+			RedundantSchemaName: indexMetadata.SchemaName,
+			RedundantTableName:  tableName,
+			RedundantIndexName:  indexMetadata.ObjectName,
+		}
+		key := tempRedundantInfo.GetRedundantIndexCatalogObjectName()
```

Comment:
instead to utilize this proper naming format you could use the thing used internally in this struct 
```
indexSqlName := sqlname.NewObjectNameQualifiedWithTableName(sourceDBType, "", indexMetadata.ObjectName, indexMetadata.SchemaName, tableName)
key := indexSqlName.Qualified.Unquoted
```
there is an `sqlname` package we have that handles the naming properly internally for objects

Reply:
incorporated this review comment as well.